docs/sdks/server-sdks/node.md

```diff
@@ -31,40 +31,11 @@ npm install @eppo/node-server-sdk
-
-```javascript
-import { init } from "@eppo/node-server-sdk";
-
-await init({
-  apiKey: "<API_KEY>",
-  assignmentLogger,
-});
-```
-
-After initialization, the SDK begins polling Eppo’s API at regular intervals to retrieve the most recent experiment configurations such as variation values and traffic allocation. The SDK stores these configurations in memory so that assignments thereafter are effectively instant. For more information, see the [architecture overview](/sdks/overview) page.
-
-If you are using the SDK for experiment assignments, make sure to pass in an assignment logging callback (see [section](#define-an-assignment-logger-experiment-assignment-only) below).
-
-### Initialization options
-
-How the SDK fetches experiment configurations is configurable via additional optional initialization options:
-
-| Option | Description | Default |
-| ------ | ----------- | ------- | 
-| **`requestTimeoutMs`** (number) | Timeout in milliseconds for HTTPS requests for the experiment configurations. | `5000` |
-| **`numInitialRequestRetries`** (number) | Number of _additional_ times the initial configurations request will be attempted if it fails. This is the request typically synchronously waited (via `await`) for completion. A small wait will be done between requests. | `1` |
-| **`pollAfterFailedInitialization`** (boolean) | Poll for new configurations even if the initial configurations request failed. | `false` |
-| **`throwOnFailedInitialization`** (boolean) | Throw an error (reject the promise) if unable to fetch initial configurations during initialization. | `true` |
-| **`numPollRequestRetries`** (number) | If polling for updated configurations after initialization, the number of additional times a request will be attempted before giving up. Subsequent attempts are done using an exponential backoff. | `7` |
-
-### Define an assignment logger (experiment assignment only)
+## 2. Define an assignment logger
 
 If you are using the Eppo SDK for experiment assignment (i.e randomization), pass in a callback logging function to the `init` function on SDK initialization. The SDK invokes the callback to capture assignment data whenever a variation is assigned.
 
-The code below illustrates an example implementation of a logging callback using Segment. You could also use your own logging system, the only requirement is that the SDK receives a `logAssignment` function. Here we define an implementation of the Eppo `IAssignmentLogger` interface containing a single function named `logAssignment`:
+The code below illustrates an example implementation of a logging callback to Segment. You could also use your own logging system, the only requirement is that the SDK receives a `logAssignment` function. Here we define an implementation of the Eppo `IAssignmentLogger` interface containing a single function named `logAssignment`:
 
 ```javascript
```

Comment:
Can we do a similar `<Tabs>`  thing like we do in the "Install the SDK" section? And then default to the console logger from the example below:

```
const assignmentLogger: IAssignmentLogger = {
  logAssignment(assignment) {
        console.log(assignment)
  },
};
```

Reply:
We can also add other examples from this page: https://docs.geteppo.com/sdks/event-logging

Reply:
i did have that originally, but wasn't sure if it'd be overkill since it's linked to this other doc. I'll add in the tabs for examples here

Reply:
updated